azkaban-common/src/main/java/azkaban/utils/EmailMessage.java

```diff
@@ -287,4 +287,13 @@ public EmailMessage println(Object str) {
 
     return this;
   }
+
+  public String getBody() {
```

Comment:
This method is for unit testing only, right?

How about making it package private? This way the readers would know that this is not a public API outside this package and would reduce the search space.